src/client/components/Table/Table.tsx

```diff
@@ -83,9 +99,9 @@ export default function Table({
         </thead>
       )}
       <tbody>
-        {items.map((item: Unshaped, index) => (
+        {items.map((item, index) => (
           <tr
-            key={item.id || `${titleKey}-${index}`}
+            key={item.id || `${String(titleKey)}-${index}`}
```

Comment:
Als `titleKey` altijd een string is, hoef je dit niet te doen. Bovendien wordt titleKey als string geïnterpoleerd doordat je de template string gebruikt.

Reply:
Typescript geeft aan dat `titleKey` in dit geval ook een Symbol kan zijn (omdat we een generic gebruiken). Daarom is de expliciete conversie hier nodig.

Reply:
Weird. Unshaped zegt duidelijk : `{ [key: string]: any }`..